nvflare/private/fed/server/server_json_config.py

```diff
@@ -138,8 +138,8 @@ def _get_all_workflows_ids(self):
             ids.append(t.id)
         return ids
 
-    def authorize_and_build_component(self, config_dict, config_ctx, node):
-        t = super().authorize_and_build_component(config_dict, config_ctx, node)
+    def build_component(self, config_dict):
+        t = super().build_component(config_dict)
```

Comment:
since it is main branch,
should we be using `authorize_and_build_component`?

Reply:
same as above.